tests/test_models.py

```diff
@@ -7,7 +7,20 @@
 
 def replace_dynamic(data: dict) -> dict:
     """Replace dynamic fields with their type name."""
-    for key in ["id", "uuid", "dbnode_id", "user_id", "mtime", "ctime"]:
+    for key in [
+        "id",
+        "uuid",
+        "dbnode_id",
+        "user_id",
+        "mtime",
+        "ctime",
+        "hostname",
+        "name",
+        "scheduler_type",
```

Comment:
Thanks @NinadBhat, I don't think you need to override all these fields though, i.e. they will always, deterministically, be the same value. As opposed to uuids which are randomly generated, and id, which are created by PostgreSQL and dependent on previous tests run (since its id "counter" does not get reset when you reset the database in between tests)